### ISSUE TITLE
Inference on a remote server  / models and streaming assets -- resources on a remote server

### ISSUE THREAD

**Issue by sobhanb-eth** [homuler/MediaPipeUnityPlugin#938]
### Plugin Version or Commit ID

v10.1

### Unity Version

2021.3.27

### Your Host OS

Windows

### Target Platform

Android, iOS

### Description

hi @homuler Is there a way to put the streaming assets / resources on a remote server so that the process can be handled there?
basically what i am aiming is to have inference on a remote server, is that even possible?
the reason is i am trying to train my own models and have them on a remote server.

if it is possible can you please tell me how?

is there a way to have the streaming assets and resources run from a server? how?

Thanks

**Comment by homuler** [comment]
If you're running it on a remote server, it seems that you don't need to worry about things like `StreamingAssets`, and you can simply set up a MediaPipe server.
In other words, I believe there is little need to use this plugin.

**Comment by sobhanb-eth** [comment]
Thanks for the response @homuler , I am not running it on a server, i was asking the question of how and if that is possible!
because i am making some gamified stuff with Mediapipe, but honestly I have no idea how to run it on a remote server, but want to go down that road, so if there is a guideline, please shed some light on my path.

on the other hand, is there a way to load the StreamingAssets from a remote location, I have watched a youtube video where someone ran a quick python server and everything was on the python side and it was connected to the unity editor and they could communicate to each other and it was fast (i mean it was a local server anyhow).

I was wondering if that can be the case with this plugin.

Thanks

**Comment by homuler** [comment]
> I am not running it on a server

I wrote `If you're running it on a remote server` to mean if you intend to run it on a server.
If that's the case, instead of using this plugin, you should run MediaPipe itself on the server. If you're unsure about how to do that, please ask elsewhere for guidance.

> on the other hand, is there a way to load the StreamingAssets from a remote location

Of course, you can read the required resources from the remote server (though we don't call them `StreamingAssets`).
Just to clarify, the need for downloading resources is only applicable when you want to run MediaPipe locally. If you are running a Python server, there is no need for it, since the client don't need to read the model file.

> I was wondering if that can be the case with this plugin.

In conclusion, this plugin is intended for running MediaPipe on the client-side, and there is no need to use it if you are running it on the server-side.
